swarm/src/test/resources/docker/swarm.tests.bom

```diff
@@ -396,6 +396,7 @@ brooklyn.catalog:
         swarm.port: 4000
         docker.recovery.stabilizationDelay: 10s
         docker.recovery.failOnRecurringFailuresInThisDuration: 5m
+        swarm.sharedsecuritygroup.create: true
```

Comment:
I think so. swarm.sharedsecuritygroup.create is a brooklyn.parameter that is defaulted to 'true' in the main swarm blueprint. 

Setting it to true in the test ensure that we test what a user will get by default.